web_timeline: fix group orders

jcdrubay opened the pull request:
Without this change, the groups are ordered kind of randomly because the javascript operation `-` does not work between 2 strings.

docmfried committed:
Added new form field web timepicker widget for Odoo Version 9.0

docmfried committed:
[ADD] Added new form field web timepicker widget for Odoo Version 9.0

docmfried committed:
[FIX]: Formal license and Travis LINT errors
[FIX]: Renaming files according to OCA standards
[FIX]: Change internal persistance form char to float
[FIX]: Reorder jquery plugin to lib directory
[ADD]: Form field record defintion can override timepicker plugin options

docmfried committed:
[FIX]: More TravisCI LINT errors
[FIX]: Missing semicolon in web_widget_timepicker.js

docmfried committed:
[FIX]: More TravisCI LINT errors
[FIX]: Replace eval() function in web_widget_timepicker.js
[FIX]: Change options to data-options in field record defintion

docmfried committed:
[FIX]: Remove needless comments in __openerp__.py
[FIX]: Cleanup user provided timepicker options initialization and handling

docmfried committed:
[FIX]: Change README.rst according to OCA template
[IMP]: New module icon
[ADD]: Screenshot / images to give an visual overview

pedrobaeza committed:
[MIG] Make modules uninstallable

pedrobaeza committed:
[MIG] Rename manifest files

smurf-U committed:
[10.0][MIG]migrate web_widget_timepicker

cubells committed:
[ADD] web_widget_digitized_signature

pedrobaeza committed:
[IMP] web_widget_digitized_signature: Add tests

oca-transbot committed:
OCA Transbot updated translations from Transifex

cubells committed:
[FIX] Incorrect field name to track signature (#691)

eLBati committed:
porting  web_widget_digitized_signature to 10.0

oca-transbot committed:
OCA Transbot updated translations from Transifex

hbrunn committed:
[FIX] RST syntax

oca-travis committed:
[UPD] Update web_widget_timepicker.pot

Tardo committed:
[11.0][MIG] web_widget_timepicker

Tardo committed:
[11.0][FIX] web_responsive: Loading indicator

jcdrubay committed:
web_timeline: fix group orders

jcdrubay commented:
The fix seems to work with Firefox but not with Chrome :/

jcdrubay committed:
web_timeline: fix order of groups for Chrome using a proper sort function

Tardo committed:
[11.0][FIX] web_export_view: Issue #1099

Tardo committed:
[11.0][IMP] web_export_view: Split readme

Tardo committed:
[11.0][IMP] web_export_view: Linter

tarteo reviewed (commented):
Is `grp1.content` and `grp2.content` always a string? If this is the case you can use https://www.w3schools.com/jsref/jsref_localecompare.asp

oca-travis committed:
[UPD] Update web_widget_timepicker.pot

cubells committed:
[MIG] web_widget_digitized_signature: Migration to 11.0

tarteo committed:
[FIX] web_widget_x2many_2d_matrix: Use widget for formatting aggregates

pedrobaeza committed:
Update __manifest__.py

oca-travis committed:
[UPD] Update web_widget_digitized_signature.pot

oca-travis committed:
[UPD] Update web_export_view.pot

oca-transbot committed:
Update translation files

Updated by "Update PO files to match POT (msgmerge)" hook in Weblate.

Translation: web-11.0/web-11.0-web_export_view
Translate-URL: https://translation.odoo-community.org/projects/web-11-0/web-11-0-web_export_view/

GSLabIt committed:
Translated using Weblate (Italian)

Currently translated at 66.7% (2 of 3 strings)

Translation: web-11.0/web-11.0-web_widget_x2many_2d_matrix
Translate-URL: https://translation.odoo-community.org/projects/web-11-0/web-11-0-web_widget_x2many_2d_matrix/it/

GSLabIt committed:
Translated using Weblate (Italian)

Currently translated at 100.0% (3 of 3 strings)

Translation: web-11.0/web-11.0-web_widget_x2many_2d_matrix
Translate-URL: https://translation.odoo-community.org/projects/web-11-0/web-11-0-web_widget_x2many_2d_matrix/it/

sbidoul committed:
[FIX] title level in readme fragment

OCA-git-bot committed:
[ADD] icon.png

OCA-git-bot committed:
[UPD] README.rst

OCA-git-bot committed:
[ADD] setup.py

jcdrubay committed:
Refactor code to use str1.localeCompare(str2);

jcdrubay commented:
Sorry for the mess in the history, I created this new PR with a clean history: https://github.com/OCA/web/pull/1283
